tedana/workflows/tedana.py

```diff
@@ -2,23 +2,17 @@
 Run the "canonical" TE-Dependent ANAlysis workflow.
 """
 import os
-
-os.environ['MKL_NUM_THREADS'] = '1'
-os.environ['NUMEXPR_NUM_THREADS'] = '1'
-os.environ['OMP_NUM_THREADS'] = '1'
-os.environ['VECLIB_MAXIMUM_THREADS'] = '1'
-os.environ['OPENBLAS_NUM_THREADS'] = '1'
-
+import os.path as op
 import shutil
 import logging
-import os.path as op
-from glob import glob
 import datetime
+from glob import glob
```

Comment:
Any reason for the reordering of imports here?

Reply:
I moved os.path up to right below os to group imports from the same package together. I moved glob down to match our general pattern of increasing length (grouped first by built-in, third-party, then local imports).

Reply:
Gotcha! I generally use alphabetical so I think that threw me off. I see now that none of the rest are alphabetical either so :man_shrugging: Works for me!